src/ext/php7/handlers_phpredis.c

```diff
@@ -0,0 +1,185 @@
+#include "handlers_internal.h"
```

Comment:
I totally missed this the first time around: We'll need to add `phpredis` to the [`ddtrace_integration_name` enum](https://github.com/DataDog/dd-trace-php/blob/c8474ce/src/ext/integrations/integrations.h#L10-L30) as well as the [`ddtrace_integrations` array](https://github.com/DataDog/dd-trace-php/blob/c8474ceee6e3a50945a94ae80fc6e418acde383b/src/ext/integrations/integrations.c#L7-L27) otherwise the integration-specific config will always use the defaults for `phpredis`.

I should add a test to check this every time we add a new integration.

Reply:
Test for integrations at the extension level are added in #985.